### ISSUE TITLE
Finetune a network and NO FIGURE (accuracy and loss) is shown

### ISSUE THREAD

**Issue by yyyreal** [NVIDIA/DIGITS#2017]
I was trying to finetune the NIN (network in network) network in digits, using public database.
The finetune progress worked fine and it got high accuracy.
But I notice that in the web page, there should have been a figure above the figure of learning_rate， shown the change of the loss and accuracy thought training process.

![screen shot 2018-04-09 at 17 40 25](https://user-images.githubusercontent.com/13974360/38490958-1c86985e-3c1d-11e8-8490-07b812e1a508.png)

Useful information such as loss and accuracy this project can only be retrieved from the raw caffe output (which proves the validity of my configuration)

Went thought the [official fine-tune demo ](https://github.com/NVIDIA/DIGITS/blob/master/examples/fine-tuning/README.md) ( fine tune lenet network to recognize odd or even numbers), I could see the figure showing the loss and accuracy. The output of digits is shown below.

![screen shot 2018-04-09 at 17 47 02](https://user-images.githubusercontent.com/13974360/38491369-3c67dd26-3c1e-11e8-86f4-485672b783b4.png)

Could anyone help to me to solve this issue? Thanks a lot.
